test/e2e/framework/util.go

```diff
@@ -3683,6 +3683,7 @@ func ParseKVLines(output, key string) string {
 	return ""
 }
 
+// RestartKubeProxy restarts a kube-proxy on the given host.
```

Comment:
Thanks @atoato88 
added some minor comments.
```suggestion
// RestartKubeProxy restarts kube-proxy on the given host.
```

Reply:
done.